2023-02-26 01:04:59,258 - INFO-[31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5000 LAPTOP-QLAPL0IL
2023-02-26 01:04:59,259 - INFO-[33mPress CTRL+C to quit[0m LAPTOP-QLAPL0IL
2023-02-26 01:05:09,806 - INFO-127.0.0.1 - - [26/Feb/2023 01:05:09] "GET / HTTP/1.1" 200 - LAPTOP-QLAPL0IL
2023-02-26 01:05:09,920 - INFO-127.0.0.1 - - [26/Feb/2023 01:05:09] "[36mGET /static/css/main.css HTTP/1.1[0m" 304 - LAPTOP-QLAPL0IL
2023-02-26 01:05:14,958 - INFO-Post Method called LAPTOP-QLAPL0IL
2023-02-26 01:05:16,530 - INFO-read html data LAPTOP-QLAPL0IL
2023-02-26 01:05:18,532 - INFO-retrive the product link LAPTOP-QLAPL0IL
2023-02-26 01:05:19,631 - INFO-Response https://www.flipkart.com/mirana-usb-rechargeable-racing-rc-car-high-speed-remote-control-toy/p/itm4eb1bbb56858c?pid=RCTGGSKGZ6MJPZKG&lid=LSTRCTGGSKGZ6MJPZKGV3GXWN&marketplace=FLIPKART&q=rc&store=tng%2F56a%2Ffq8&srno=s_1_5&otracker=search&fm=organic&iid=a0c5bec9-1e02-4f59-93b5-16a712510396.RCTGGSKGZ6MJPZKG.SEARCH&ppt=None&ppn=None&ssid=495ilribtc0000001677353715858&qH=ff78648be52a4e79 LAPTOP-QLAPL0IL
2023-02-26 01:05:19,824 - ERROR-Exception on /review [POST] LAPTOP-QLAPL0IL
Traceback (most recent call last):
  File "C:\Users\praja\OneDrive\Desktop\scrapping\app.py", line 66, in review
    cust_name = i.div.div.find_all('p', {'class': '_2sc7ZR _2V5EHH"'})[0].text
IndexError: list index out of range

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\praja\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 2528, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\praja\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1825, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\praja\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1823, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\praja\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1799, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
  File "C:\Users\praja\OneDrive\Desktop\scrapping\app.py", line 70, in review
    logging.erro(e)
AttributeError: module 'logging' has no attribute 'erro'
2023-02-26 01:05:19,836 - INFO-127.0.0.1 - - [26/Feb/2023 01:05:19] "[35m[1mPOST /review HTTP/1.1[0m" 500 - LAPTOP-QLAPL0IL
2023-02-26 01:06:32,194 - INFO-[31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5000 LAPTOP-QLAPL0IL
2023-02-26 01:06:32,194 - INFO-[33mPress CTRL+C to quit[0m LAPTOP-QLAPL0IL
2023-02-26 01:06:37,172 - INFO-127.0.0.1 - - [26/Feb/2023 01:06:37] "GET / HTTP/1.1" 200 - LAPTOP-QLAPL0IL
2023-02-26 01:06:37,296 - INFO-127.0.0.1 - - [26/Feb/2023 01:06:37] "[36mGET /static/css/main.css HTTP/1.1[0m" 304 - LAPTOP-QLAPL0IL
2023-02-26 01:06:40,414 - INFO-Post Method called LAPTOP-QLAPL0IL
2023-02-26 01:06:42,317 - INFO-read html data LAPTOP-QLAPL0IL
2023-02-26 01:06:44,416 - INFO-retrive the product link LAPTOP-QLAPL0IL
2023-02-26 01:06:45,689 - INFO-Response https://www.flipkart.com/mirana-usb-rechargeable-racing-rc-car-high-speed-remote-control-toy/p/itm4eb1bbb56858c?pid=RCTGGSKGZ6MJPZKG&lid=LSTRCTGGSKGZ6MJPZKGV3GXWN&marketplace=FLIPKART&q=rc&store=tng%2F56a%2Ffq8&srno=s_1_5&otracker=search&fm=organic&iid=bf494cca-c1d3-47b6-973f-093f11870513.RCTGGSKGZ6MJPZKG.SEARCH&ppt=None&ppn=None&ssid=w81jbwws9s0000001677353801962&qH=ff78648be52a4e79 LAPTOP-QLAPL0IL
2023-02-26 01:06:45,878 - ERROR-Exception on /review [POST] LAPTOP-QLAPL0IL
Traceback (most recent call last):
  File "C:\Users\praja\OneDrive\Desktop\scrapping\app.py", line 66, in review
    cust_name = i.div.div.find_all('p', {'class': '_2sc7ZR _2V5EHH'})[0].text
AttributeError: 'NoneType' object has no attribute 'div'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\praja\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 2528, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\praja\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1825, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\praja\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1823, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\praja\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1799, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
  File "C:\Users\praja\OneDrive\Desktop\scrapping\app.py", line 70, in review
    logging.erro(e)
AttributeError: module 'logging' has no attribute 'erro'
2023-02-26 01:06:45,897 - INFO-127.0.0.1 - - [26/Feb/2023 01:06:45] "[35m[1mPOST /review HTTP/1.1[0m" 500 - LAPTOP-QLAPL0IL
2023-02-26 01:12:20,365 - INFO-[31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5000 LAPTOP-QLAPL0IL
2023-02-26 01:12:20,365 - INFO-[33mPress CTRL+C to quit[0m LAPTOP-QLAPL0IL
2023-02-26 01:12:26,866 - INFO-127.0.0.1 - - [26/Feb/2023 01:12:26] "GET / HTTP/1.1" 200 - LAPTOP-QLAPL0IL
2023-02-26 01:12:26,970 - INFO-127.0.0.1 - - [26/Feb/2023 01:12:26] "[36mGET /static/css/main.css HTTP/1.1[0m" 304 - LAPTOP-QLAPL0IL
2023-02-26 01:12:30,630 - INFO-Post Method called LAPTOP-QLAPL0IL
2023-02-26 01:12:32,557 - INFO-read html data LAPTOP-QLAPL0IL
2023-02-26 01:12:34,237 - INFO-retrive the product link LAPTOP-QLAPL0IL
2023-02-26 01:12:35,084 - INFO-Response https://www.flipkart.com/mirana-usb-rechargeable-racing-rc-car-high-speed-remote-control-toy/p/itm4eb1bbb56858c?pid=RCTGGSKGZ6MJPZKG&lid=LSTRCTGGSKGZ6MJPZKGV3GXWN&marketplace=FLIPKART&q=rc&store=tng%2F56a%2Ffq8&srno=s_1_5&otracker=search&fm=organic&iid=28cdca4c-9b6f-4dee-aa5a-f715d19929d5.RCTGGSKGZ6MJPZKG.SEARCH&ppt=None&ppn=None&ssid=20fmyre6g00000001677354151718&qH=ff78648be52a4e79 LAPTOP-QLAPL0IL
2023-02-26 01:12:35,233 - ERROR-Exception on /review [POST] LAPTOP-QLAPL0IL
Traceback (most recent call last):
  File "C:\Users\praja\OneDrive\Desktop\scrapping\app.py", line 66, in review
    cust_name = i.find_all("p", {"class": "_2sc7ZR _2V5EHH"})[0].text
IndexError: list index out of range

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\praja\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 2528, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\praja\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1825, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\praja\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1823, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\praja\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1799, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
  File "C:\Users\praja\OneDrive\Desktop\scrapping\app.py", line 70, in review
    logging.erro(e)
AttributeError: module 'logging' has no attribute 'erro'
2023-02-26 01:12:35,270 - INFO-127.0.0.1 - - [26/Feb/2023 01:12:35] "[35m[1mPOST /review HTTP/1.1[0m" 500 - LAPTOP-QLAPL0IL
2023-02-26 01:14:35,000 - INFO-[31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5000 LAPTOP-QLAPL0IL
2023-02-26 01:14:35,000 - INFO-[33mPress CTRL+C to quit[0m LAPTOP-QLAPL0IL
2023-02-26 01:14:41,805 - INFO-127.0.0.1 - - [26/Feb/2023 01:14:41] "GET / HTTP/1.1" 200 - LAPTOP-QLAPL0IL
2023-02-26 01:14:41,895 - INFO-127.0.0.1 - - [26/Feb/2023 01:14:41] "[36mGET /static/css/main.css HTTP/1.1[0m" 304 - LAPTOP-QLAPL0IL
2023-02-26 01:14:48,738 - INFO-Post Method called LAPTOP-QLAPL0IL
2023-02-26 01:14:50,641 - INFO-read html data LAPTOP-QLAPL0IL
2023-02-26 01:14:52,210 - INFO-retrive the product link LAPTOP-QLAPL0IL
2023-02-26 01:14:53,691 - INFO-Response https://www.flipkart.com/india-gate-brown-rice-weight-watchers-special-basmati/p/itmd895fc7bea89c?pid=RICFUTBUNHTNNTKG&lid=LSTRICFUTBUNHTNNTKGLZAFJI&marketplace=FLIPKART&q=rice&store=eat%2Fyul&srno=s_1_5&otracker=search&fm=organic&iid=ef4cb861-824b-43ff-b474-ba27d3ff238a.RICFUTBUNHTNNTKG.SEARCH&ppt=None&ppn=None&ssid=0zsop0tsrk0000001677354289827&qH=3767b450824877f2 LAPTOP-QLAPL0IL
2023-02-26 01:14:53,807 - ERROR-Exception on /review [POST] LAPTOP-QLAPL0IL
Traceback (most recent call last):
  File "C:\Users\praja\OneDrive\Desktop\scrapping\app.py", line 66, in review
    cust_name = i.find_all("p", {"class": "_2sc7ZR _2V5EHH"})[0].text
IndexError: list index out of range

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\praja\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 2528, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\praja\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1825, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\praja\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1823, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\praja\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1799, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
  File "C:\Users\praja\OneDrive\Desktop\scrapping\app.py", line 70, in review
    logging.error(e)
UnboundLocalError: local variable 'e' referenced before assignment
2023-02-26 01:14:53,820 - INFO-127.0.0.1 - - [26/Feb/2023 01:14:53] "[35m[1mPOST /review HTTP/1.1[0m" 500 - LAPTOP-QLAPL0IL
2023-02-26 01:15:46,169 - INFO-[31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5000 LAPTOP-QLAPL0IL
2023-02-26 01:15:46,169 - INFO-[33mPress CTRL+C to quit[0m LAPTOP-QLAPL0IL
2023-02-26 01:15:53,272 - INFO-Post Method called LAPTOP-QLAPL0IL
2023-02-26 01:15:56,322 - INFO-read html data LAPTOP-QLAPL0IL
2023-02-26 01:15:57,783 - INFO-retrive the product link LAPTOP-QLAPL0IL
2023-02-26 01:15:59,230 - INFO-Response https://www.flipkart.com/india-gate-brown-rice-weight-watchers-special-basmati/p/itmd895fc7bea89c?pid=RICFUTBUNHTNNTKG&lid=LSTRICFUTBUNHTNNTKGLZAFJI&marketplace=FLIPKART&q=rice&store=eat%2Fyul&srno=s_1_5&otracker=search&fm=organic&iid=9ef494dd-87a1-4702-bd61-52b2153164c4.RICFUTBUNHTNNTKG.SEARCH&ppt=None&ppn=None&ssid=dtvkykouy80000001677354354590&qH=3767b450824877f2 LAPTOP-QLAPL0IL
2023-02-26 01:15:59,340 - ERROR-list index out of range LAPTOP-QLAPL0IL
2023-02-26 01:15:59,372 - INFO-127.0.0.1 - - [26/Feb/2023 01:15:59] "POST /review HTTP/1.1" 200 - LAPTOP-QLAPL0IL
2023-02-26 01:15:59,432 - INFO-127.0.0.1 - - [26/Feb/2023 01:15:59] "[33mGET /style.css HTTP/1.1[0m" 404 - LAPTOP-QLAPL0IL
2023-02-26 01:15:59,508 - INFO-127.0.0.1 - - [26/Feb/2023 01:15:59] "[36mGET /static/css/style.css HTTP/1.1[0m" 304 - LAPTOP-QLAPL0IL
2023-02-26 01:16:15,912 - INFO-Post Method called LAPTOP-QLAPL0IL
2023-02-26 01:16:17,383 - INFO-read html data LAPTOP-QLAPL0IL
2023-02-26 01:16:18,798 - INFO-retrive the product link LAPTOP-QLAPL0IL
2023-02-26 01:16:19,678 - INFO-Response https://www.flipkart.com/mirana-usb-rechargeable-racing-rc-car-high-speed-remote-control-toy/p/itm4eb1bbb56858c?pid=RCTGGSKGZ6MJPZKG&lid=LSTRCTGGSKGZ6MJPZKGV3GXWN&marketplace=FLIPKART&q=rc&store=tng%2F56a%2Ffq8&srno=s_1_5&otracker=search&fm=organic&iid=fd656990-f049-447d-af38-66d164e72663.RCTGGSKGZ6MJPZKG.SEARCH&ppt=None&ppn=None&ssid=0ud4f4wckg0000001677354376765&qH=ff78648be52a4e79 LAPTOP-QLAPL0IL
2023-02-26 01:16:19,842 - ERROR-list index out of range LAPTOP-QLAPL0IL
2023-02-26 01:16:19,850 - INFO-127.0.0.1 - - [26/Feb/2023 01:16:19] "POST /review HTTP/1.1" 200 - LAPTOP-QLAPL0IL
2023-02-26 01:16:19,913 - INFO-127.0.0.1 - - [26/Feb/2023 01:16:19] "[33mGET /style.css HTTP/1.1[0m" 404 - LAPTOP-QLAPL0IL
2023-02-26 01:16:19,943 - INFO-127.0.0.1 - - [26/Feb/2023 01:16:19] "[36mGET /static/css/style.css HTTP/1.1[0m" 304 - LAPTOP-QLAPL0IL
2023-02-26 01:20:42,499 - INFO-[31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5000 192.168.43.91
2023-02-26 01:20:42,500 - INFO-[33mPress CTRL+C to quit[0m 192.168.43.91
2023-02-26 01:21:00,295 - INFO-127.0.0.1 - - [26/Feb/2023 01:21:00] "GET / HTTP/1.1" 200 - 192.168.43.91
2023-02-26 01:21:00,407 - INFO-127.0.0.1 - - [26/Feb/2023 01:21:00] "[36mGET /static/css/style.css HTTP/1.1[0m" 304 - 192.168.43.91
2023-02-26 01:21:00,425 - INFO-127.0.0.1 - - [26/Feb/2023 01:21:00] "[36mGET /static/css/main.css HTTP/1.1[0m" 304 - 192.168.43.91
2023-02-26 01:21:08,175 - INFO-Post Method called 192.168.43.91
2023-02-26 01:21:10,151 - INFO-read html data 192.168.43.91
2023-02-26 01:21:11,906 - INFO-retrive the product link 192.168.43.91
2023-02-26 01:21:13,596 - INFO-Response https://www.flipkart.com/india-gate-brown-rice-weight-watchers-special-basmati/p/itmd895fc7bea89c?pid=RICFUTBUNHTNNTKG&lid=LSTRICFUTBUNHTNNTKGLZAFJI&marketplace=FLIPKART&q=rice&store=eat%2Fyul&srno=s_1_5&otracker=search&fm=organic&iid=d55d5c79-2791-448a-9e0e-8deb95ea97cf.RICFUTBUNHTNNTKG.SEARCH&ppt=None&ppn=None&ssid=9r1w0uhoxc0000001677354669205&qH=3767b450824877f2 192.168.43.91
2023-02-26 01:21:13,713 - ERROR-list index out of range 192.168.43.91
2023-02-26 01:21:13,725 - INFO-127.0.0.1 - - [26/Feb/2023 01:21:13] "POST /review HTTP/1.1" 200 - 192.168.43.91
2023-02-26 01:21:13,775 - INFO-127.0.0.1 - - [26/Feb/2023 01:21:13] "[33mGET /style.css HTTP/1.1[0m" 404 - 192.168.43.91
2023-02-26 01:21:13,795 - INFO-127.0.0.1 - - [26/Feb/2023 01:21:13] "[36mGET /static/css/style.css HTTP/1.1[0m" 304 - 192.168.43.91
